.github/workflows/main.yml

```diff
@@ -218,4 +218,5 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 18
-      - run: cargo test -p cargo-risczero -F docker
+      - run: cargo test -p risc0-build
+      - run: cargo test -F default -F profiler -F test-exact-cycles
```

Comment:
I'd probably limit this to `risc0-zkvm` to help with build times

Reply:
thanks for catching this!